<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>39</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_ajout">
     <attribute name="title">
      <string>Ajout d'employé</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_cin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dateE">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numT">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_salaire">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_abs">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prenom">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_dateE">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>220</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE D'EMB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_salaire">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SALAIRE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_numT">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NUM TEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_abs">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ABSENCE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>310</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>AJOUTER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_consul">
     <attribute name="title">
      <string>Consulter les employées</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>371</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_supp">
     <attribute name="title">
      <string>Supprimer Employer</string>
     </attribute>
     <widget class="QLabel" name="label_cinsupp">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>SUPPRIMER</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cinsupp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
